README.md

```diff
@@ -51,6 +51,18 @@ Prometheus ideas:
 
 #### Refactor the APIs for better readability and less maintenance overhead
```

Comment:
this one can be removed now as we decided to test swagger with the alertmanager and only than maybe use it for Prometheus as well so it is on hold for now.

Not sure what is the progress of this for the Alertmanager, maybe @mxinden can update what is the progress and if any help is needed for the alertmanager.

Reply:
Alertmanagers API v2, generated based on a Swagger spec, will be part of v0.16.0. The release is planned for next week: https://github.com/prometheus/alertmanager/pull/1714

As this has only been on the master branch so far, not many have been able to test it out.

Reply:
Thanks for the update Max